lib/Backends/NNPI/NNPI.cpp

```diff
@@ -1154,10 +1160,12 @@ Error NNPIBackend::bindContexts(
     nnpiDM->addPlaceholderUsageCount(cb.networkName, phUsage);
   }
 
-  for (const auto &usage : phUsage) {
+  for (auto &usage : phUsage) {
     LOG_IF_NOT_RETURN_LLVMERROR(
         usage.second.numWriters < 2,
         "Multiple writes to the same placeholder not suported");
+    usage.second.disableP2P = !enableP2P;
```

Comment:
Does this need to be per PH? Looks like the only place it's used is in an OR statement with the Device level setting.

Reply:
yes it's controlled at the PH level. see NNPIResource.cpp:180 in this PR.